### ISSUE TITLE
doesnt actually log anything?

### ISSUE THREAD

**Issue by aep** [ic2hrmk/promtail#3]
Hi,
i'm trying this with loki 2.4.2 and it doesnt appear to actually write anything.
promtail works fine.

but this api doesnt work either:  https://grafana.com/docs/loki/latest/api/#examples-8

i guess protobuf is required now?

**Comment by ic2hrmk** [comment]
Hi @aep, I don't support this repo anymore. Feel free to raise a PR if you up to

**Comment by gatlee21** [comment]
Yeah, commenting to agree with @aep. I had to run the example file 5 times just for one Info log to appear in Loki which makes this unreliable. I'm using Loki 2.4.0 btw.

I was really excited when I first found this project as it supports labels per log entry, unlike [Afiskons](https://github.com/afiskon/promtail-client). Hopefully, there's still enough interest in this project that we could get this fixed @ic2hrmk?
